src/com/softwaret/kpdf/util/extension/ApplicationConfigExtension.kt

```diff
@@ -6,14 +6,26 @@ import com.auth0.jwt.algorithms.Algorithm
 import com.softwaret.kpdf.model.inline.Milliseconds
 import io.ktor.config.ApplicationConfig
 
+val ApplicationConfig.algorithmPath
```

Comment:
Widzę, że jest to wykorzystywane testach, ale dlaczego nie zrobimy nazwy tego pola po prostu jako `private const val` ?

Reply:
? :(